addons/mail/models/mail_message.py

```diff
@@ -868,17 +879,25 @@ def _message_reaction(self, content, action):
             self.env["mail.message.reaction"].create(create_values)
         if action == "remove" and reaction:
             reaction.unlink()
+        # fill the store to use for non logged in portal users in mail_message_add_reaction()
+        self._bus_send_reaction_group(content, store)
+        # send the reaction group to bus for logged in users
         self._bus_send_reaction_group(content)
 
-    def _bus_send_reaction_group(self, content):
+    def _bus_send_reaction_group(self, content, store=None):
```

Comment:
no need for param

Reply:
I pass the store in line 883 in calling this function. Why no need for param?

Reply:
Ok. got it